### ISSUE TITLE
Rule help text override is ignored

### ISSUE THREAD

**Issue by marceldegraaf** [prometheus/cloudwatch_exporter#11]
According to [the code](https://github.com/prometheus/cloudwatch_exporter/blob/master/src/main/java/io/prometheus/cloudwatch/CloudWatchCollector.java#L103-L105) it's possible to add a `help` tag to a JSON metrics object, which – I guess – should then be used in the metrics output for Prometheus. For example:

```
{
  "metrics": [
    {
      "help": "test",
      "aws_namespace": "AWS/DynamoDB",
      "aws_metric_name": "ProvisionedReadCapacityUnits",
      "aws_dimensions": [ ... ],
      "aws_dimension_select_regex": { ... }
    }
  ]
}
```

However, when compiling the final help text for Prometheus, this override [is ignored](https://github.com/prometheus/cloudwatch_exporter/blob/master/src/main/java/io/prometheus/cloudwatch/CloudWatchCollector.java#L261-L265). Instead of returning `rule.help`, a generated help string is always returned.

**Comment by brian-brazil** [comment]
Thanks for the report, that's a silly bug. I've put it on our list for the Digital Ocean Hacktoberfest as it's a nice starter project.

**Comment by marceldegraaf** [comment]
Hi @brian-brazil, any news on this? Because of this limitation we can currently not export some data to Prometheus, because the export file contains multiple identical help strings which is invalid syntax.

**Comment by brian-brazil** [comment]
> the export file contains multiple identical help strings which is invalid syntax.

That should be valid syntax, only if they're different will you get an error last time I checked.

**Comment by brian-brazil** [comment]
This was fixed in #12

**Comment by marceldegraaf** [comment]
Great, thanks!
